index.html

```diff
@@ -0,0 +1,2071 @@
+                  <a href="#"
+                    ><b class="text-body font-weight-bolder">Best Online Shops 2021</b></a
+                  >
+                  <p
+                    >Newsweek and Statista are recognizing 1,000 shops across 39
+                    categories in eight groups.</p
+                  >
+                </div>
+              </div>
+
+              <div class="card">
+                <img src="./images/2021.png" class="card-img-top" alt="2021" />
+                <div class="card-body">
+                  <a href="#"
+                    ><b class="text-body font-weight-bolder">Best Online Shops 2021</b></a
+                  >
+                  <p
+                    >Newsweek and Statista are recognizing 1,000 shops across 39
+                    categories in eight groups.</p
+                  >
+                </div>
+              </div>
+            </section>
+          </div>
+          <section
+            class="top-stories order-first d-md-none container py-4 d-flex flex-column"
+          >
+            <article
+              ><h6>TOP STORY</h6>
+              <div class="position-relative"
+                ><a href="#"
+                  ><img
+                    class="w-100 h-auto"
+                    src="./images/mcconnell.png"
+                    alt="mcconnell"
+                  />
+                  <div
+                    class="pic-tag position-absolute bg-white px-2 font-weight-bolder"
+                  >
+                    POLITICS
+                  </div>
+                </a>
+              </div>
+              <a href="#"
+                ><h3 class="text-body"
+                  >Senate GOP Unveil COVID-19 Relief Package, McConnell Demands
+                  Vote This Week</h3
+                ></a
+              >
+              <p
+                >Senate Majority Leader Mitch McConnell and the Republicans are
+                set to unveil their own COVID-19 relief package proposal and are
+                set to hold a vote on it this week.</p
+              >
+              <hr />
+              <div class="d-flex flex-row">
+                <img
+                  class="mcconnell2 mr-3"
+                  src="./images/mcconnell2.png"
+                  alt="mcconnell2"
+                />
+                <div>
+                  <a href="#"
+                    ><b class="text-body font-weight-bolder"
+                      >New 'Targeted' GOP Stimulus Excludes Second Stimulus
+                      Check, Offers Some Unemployment Relief</b
+                    ></a
+                  >
+                  <p
+                    >The $500 billion bill is Republicans' slimmest relief
+                    proposal yet, a "targeted" measure that Senate Majority
+                    Leader Mitch McConnell will try to advance in the upper
+                    chamber later this week after trying to rally GOP
+                    colleagues.</p
+                  >
```

Comment:
- [ optional ] The text should wrap around the image

 | Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/92959644-913e0080-f442-11ea-8192-d5e253995ef1.png) | ![image](https://user-images.githubusercontent.com/61479648/92959652-98650e80-f442-11ea-99b8-9b08a493a723.png) |